Lib/xmlrpc/server.py

```diff
@@ -402,13 +402,11 @@ def _dispatch(self, method, params):
                 raise Exception('method "%s" is not supported' % method)
```

Comment:
For consistency with following code I would revert the condition (this is just a style nit):
```
if func is not None:
    return ...
raise ...
```